README.md

```diff
@@ -15,16 +15,24 @@
    limitations under the License.
 --->
 
-# Eiffel
-The Eiffel framework enables technology agnostic enterprise scale continuous integration and delivery with maintained scalability, flexibility and traceability. Eiffel is based on the concept of decentralized real time messaging, both to drive the continuous integration and delivery system and to document it.
+<img src="./images/eiffel-protocol-logo.png" alt="Eiffel Protocol" width="350"/>
 
-This repository contains the Eiffel framework vocabulary, descriptions, guides and schemas along with links to relevant implementation repositories. For news, discussions and questions, please visit the [Eiffel Community Google group](https://groups.google.com/forum/#!forum/eiffel-community).
+# Eiffel Protocol
+The Eiffel protocol enables technology agnostic enterprise scale continuous integration and delivery with maintained scalability, flexibility and traceability. Eiffel is based on the concept of decentralized real time messaging, both to drive the continuous integration and delivery system and to document it.
 
-Eiffel is licensed under the [Apache License 2.0](./LICENSE).
+This repository contains the Eiffel protocol vocabulary, descriptions, guides and schemas. For implementations, implementations, architecture and community resources, visit the [Eiffel Community](https://eiffel-community.github.io).
 
-__IMPORTANT NOTICE:__ The contents of this repository currectly reflect a __DRAFT__. The Eiffel framework has been used in production within Ericsson for several years to great effect; what is presented here is a revision and evolution of that framework - an evolution that is currently ongoing. In other words, anything in this repository should be regarded as tentative and subject to change. It is published here to allow early access and trial and to solicit early feedback.
+# About this repository
+The contents of this repository are licensed under the [Apache License 2.0](./LICENSE).
 
-## Contents
+To get involved, please see [Code of Conduct](./CODE_OF_CONDUCT.md) and [contribution guidelines](./CONTRIBUTING.md).
+
+# About Eiffel
```

Comment:
I think the Eiffel Protocol above and this section 'About Eiffel' does not really make the distinction between Eiffel and the Eiffel Protocol clear enough, instead it adds to the confusion betweem them. The Eiffel Community manages a protocol and a set of implementations enabling the use of that protocol. Now, some text in 'Eiffel Protocol' above is duplicated but slightly reformulated in this 'About Eiffel' section. I think this 'About Eiffel' section is redundant and can be removed, while some of its details or phrasing could possibly be moved to the 'Eiffel Protocol' section above.

Reply:
You're right. I would solve it differently, though. The "About Eiffel" section is copy-pasted from the eiffel-community repository template, and there's a point to being consistent across repositories. Instead I removed the redundant statements in the first paragraph.

What do you think, does that work?

Reply:
Works fine